.github/workflows/e2e-tests-chrome.yml

```diff
@@ -47,7 +50,7 @@ jobs:
           CI: true
 
       - name: Run e2e tests
-        uses: cypress-io/github-action@v2
+        uses: cypress-io/github-action@v2.9.1
```

Comment:
While investigating e2e test failures, I also noticed that we're not pinning the version of the GH action. I think we should be doing that just in case there is some subtle change.